fs/rc/webgui/webgui.go

```diff
@@ -55,17 +55,25 @@ func CheckAndDownloadWebGUIRelease(checkUpdate bool, forceUpdate bool, fetchURL
 		return errors.New("Web GUI path exists, but is a file instead of folder. Please check the path " + extractPath)
 	}
 
+	// Get the latest release details
+	WebUIURL, tag, size, err := GetLatestReleaseURL(fetchURL)
+	if err != nil {
+		return errors.Wrap(err, "Error checking for web gui release update, skipping update")
+	}
+	dat, err := ioutil.ReadFile(tagPath)
+
+	// Check if current version is equal to latest release
+	tagsMatch := err == nil && string(dat) == tag
```

Comment:
I think the error handling is wrong here...

If err != nil then you'll declare that the tags don't match and you'll print the error